plugin/trino-kafka-event-listener/src/main/java/io/trino/plugin/eventlistener/kafka/KafkaEventListenerConfig.java

```diff
@@ -14,28 +14,31 @@
 
 package io.trino.plugin.eventlistener.kafka;
 
-import com.google.common.base.Splitter;
+import com.google.common.collect.ImmutableList;
 import io.airlift.configuration.Config;
 import io.airlift.configuration.ConfigDescription;
+import io.airlift.configuration.DefunctConfig;
+import io.airlift.configuration.validation.FileExists;
 import io.airlift.units.Duration;
 import io.airlift.units.MinDuration;
 import jakarta.validation.constraints.AssertTrue;
 import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
 
+import java.io.File;
 import java.util.Collections;
-import java.util.Map;
+import java.util.List;
 import java.util.Optional;
 import java.util.Set;
 
+import static com.google.common.collect.ImmutableList.toImmutableList;
 import static com.google.common.collect.ImmutableSet.toImmutableSet;
 import static java.util.Objects.requireNonNull;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
+@DefunctConfig("kafka-event-listener.client-config-overrides")
 public class KafkaEventListenerConfig
 {
-    // for the map splitter we need to only look for first = to make sure values like "key=value=1" are not split and instead passed as is (e.g. JAAS config of the form sasl.jaas.config="org.apache.kafka.common.security.scram.ScramLoginModule required username='trino_eventlistener' password='zzzzzz';")
```

Comment:
why fix to drop? :)

Reply:
if people are fine with the new config file thing I'll squash. The first commit is simply the alternative shortcut fix.